docs/migration-guide-1.25-2.0.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Migration Guide 1.25-2.0
+---
+
+Once you upgrade to Kyma 2.0, perform the manual steps described in the Migration Guide.
+
+## Security
+
+### Native Kubernetes authentication in Kyma
+
+Kyma 2.0 simplifies things and switches to the native Kubernetes authentication. As a result, we no longer support the following authentication and authorization components:
+
+- API Server Proxy
+- Console Backend Service
+- Dex
+- IAM Kubeconfig Service
+- Permission Controller
+
+To use the native Kubernetes authentication in Kyma, you need to remove the deprecated components manually.
+
+After the successful upgrade to Kyma 2.0, run the following [script](.assets/1.25-2.0-remove-deprecated-resources.sh), which uninstalls and deletes the unsupported items.
```

Comment:
We could additionally suggest also a simple command like:
curl https://raw.githubusercontent.com/kyma-project/kyma/e2c2947cb1facd6e8c3f95d04043a0a0973ce8d3/docs/assets/1.25-2.0-remove-deprecated-resources.sh | sh